### ISSUE TITLE
Data not being returned during create pages and dependencies missing

### ISSUE THREAD

**Issue by JonJamesDesign** [birkir/gatsby-source-prismic-graphql#28]
Looks like I'm not getting any data back during the create pages phase of the build process.

**gatsby-config.js:**
```
{
  resolve: `gatsby-source-prismic-graphql`,
  options: {
    repositoryName: `ihavebeenreplaced`,
    accessToken: `${process.env.PRISMIC_ACCESS_TOKEN}`,
    path: '/preview',
    previews: true,
    refetchInterval: 60,
    pages: [
      {
        type: 'home_page',
        match: '/',
        path: '/',
        component: require.resolve('./src/templates/Home/Home.jsx'),
      },
      {
        type: 'contact_page',
        match: '/contact',
        path: '/contact',
        component: require.resolve('./src/templates/Contact/Contact.jsx'),
      },
      {
        type: 'blog_article',
        match: '/blog/:uid',
        path: '/blog',
        component: require.resolve('./src/templates/BlogArticle/BlogArticle.jsx'),
      },
      {
        type: 'case_study',
        match: '/work/:uid',
        path: '/work',
        component: require.resolve('./src/templates/CaseStudy/CaseStudy.jsx'),
      },
      {
        type: 'event',
        match: '/events/:uid',
        path: '/events',
        component: require.resolve('./src/templates/Event/Event.jsx'),
      },
      {
        type: 'job_listing',
        match: '/careers/:uid',
        path: '/careers',
        component: require.resolve('./src/templates/JobListing/JobListing.jsx'),
      },
    ],
  },
},
```

I've installed the `gatsby-source-prismic-graphql` to my dependences, **package.json:**
```
"dependencies": {
...
"gatsby-source-prismic-graphql": "^3.0.5",
...
```

The error I'm getting in the console reads:
```
error Plugin gatsby-source-prismic-graphql returned an error


  TypeError: Cannot read property 'prismic' of undefined
  
  - gatsby-node.js:91 _callee$
    [GreatState]/[gatsby-source-prismic-graphql]/gatsby-node.js:91:30
  
  - util.js:16 tryCatcher
    [GreatState]/[bluebird]/js/release/util.js:16:23
  
  - promise.js:512 Promise._settlePromiseFromHandler
    [GreatState]/[bluebird]/js/release/promise.js:512:31
  
  - promise.js:569 Promise._settlePromise
    [GreatState]/[bluebird]/js/release/promise.js:569:18
  
  - promise.js:606 Promise._settlePromiseCtx
    [GreatState]/[bluebird]/js/release/promise.js:606:10
  
  - async.js:142 _drainQueueStep
    [GreatState]/[bluebird]/js/release/async.js:142:12
  
  - async.js:131 _drainQueue
    [GreatState]/[bluebird]/js/release/async.js:131:9
  
  - async.js:147 Async._drainQueues
    [GreatState]/[bluebird]/js/release/async.js:147:5
  
  - async.js:17 Immediate.Async.drainQueues [as _onImmediate]
    [GreatState]/[bluebird]/js/release/async.js:17:14
```

I also get a warning that the `gatsby-source-prismic-graphql` plugin is missing dependencies in it's `/utils` directory:
```
 ERROR  Failed to compile with 2 errors                                                                                                                                                                                                        11:57:34 AM

These dependencies were not found:

* apollo-boost in ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js
* apollo-cache-inmemory in ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js

To install them, you can run: npm install --save apollo-boost apollo-cache-inmemory
✖ ｢wdm｣: 
ERROR in ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js
Module not found: Error: Can't resolve 'apollo-boost' in '/Users/jonjames/Sites/GreatState/node_modules/gatsby-source-prismic-graphql/utils'
 @ ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js 14:19-42
 @ ./node_modules/gatsby-source-prismic-graphql/components/WrapPage.js
 @ ./node_modules/gatsby-source-prismic-graphql/gatsby-browser.js
 @ ./.cache/api-runner-browser-plugins.js
 @ ./.cache/api-runner-browser.js
 @ ./.cache/app.js
 @ multi event-source-polyfill (webpack)-hot-middleware/client.js?path=/__webpack_hmr&reload=true&overlay=false ./.cache/app

ERROR in ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js
Module not found: Error: Can't resolve 'apollo-cache-inmemory' in '/Users/jonjames/Sites/GreatState/node_modules/gatsby-source-prismic-graphql/utils'
 @ ./node_modules/gatsby-source-prismic-graphql/utils/getApolloClient.js 16:27-59
 @ ./node_modules/gatsby-source-prismic-graphql/components/WrapPage.js
 @ ./node_modules/gatsby-source-prismic-graphql/gatsby-browser.js
 @ ./.cache/api-runner-browser-plugins.js
 @ ./.cache/api-runner-browser.js
 @ ./.cache/app.js
 @ multi event-source-polyfill (webpack)-hot-middleware/client.js?path=/__webpack_hmr&reload=true&overlay=false ./.cache/app
ℹ ｢wdm｣: Failed to compile.
error Plugin gatsby-source-prismic-graphql returned an error
```

If I can provide any more information to help, please let me know.

**Comment by birkir** [comment]
Hey can you share your gatsby version?

**Comment by birkir** [comment]
I added a dependency for apollo-boost as later versions of gatsby don't include them, this should resolve your problem.

Released with version 3.0.6
